metricbeat/module/system/process/_meta/data.json

```diff
@@ -1,198 +1,46 @@
-                        "usage": {
-                            "bytes": 0,
-                            "max": {
-                                "bytes": 0
-                            }
-                        }
-                    },
-                    "mem": {
-                        "failures": 0,
-                        "limit": {
-                            "bytes": 0
-                        },
-                        "usage": {
-                            "bytes": 0,
-                            "max": {
-                                "bytes": 0
-                            }
-                        }
-                    },
-                    "memsw": {
-                        "failures": 0,
-                        "limit": {
-                            "bytes": 0
-                        },
-                        "usage": {
-                            "bytes": 0,
-                            "max": {
-                                "bytes": 0
-                            }
-                        }
-                    },
-                    "path": "/docker/d88e67bb6961a5bb70c1c1c48094c6030e43768eed91e827f437111888f9967e",
-                    "stats": {
-                        "active_anon": {
-                            "bytes": 0
-                        },
-                        "active_file": {
-                            "bytes": 0
-                        },
-                        "cache": {
-                            "bytes": 0
-                        },
-                        "hierarchical_memory_limit": {
-                            "bytes": 0
-                        },
-                        "hierarchical_memsw_limit": {
-                            "bytes": 0
-                        },
-                        "inactive_anon": {
-                            "bytes": 0
-                        },
-                        "inactive_file": {
-                            "bytes": 0
-                        },
-                        "major_page_faults": 0,
-                        "mapped_file": {
-                            "bytes": 0
-                        },
-                        "page_faults": 0,
-                        "pages_in": 0,
-                        "pages_out": 0,
-                        "rss": {
-                            "bytes": 0
-                        },
-                        "rss_huge": {
-                            "bytes": 0
-                        },
-                        "swap": {
-                            "bytes": 0
-                        },
-                        "unevictable": {
-                            "bytes": 0
-                        }
-                    }
-                },
-                "path": "/docker/d88e67bb6961a5bb70c1c1c48094c6030e43768eed91e827f437111888f9967e"
-            },
-            "cmdline": "go test -tags=integration github.com/elastic/beats/metricbeat/module/... -data",
```

Comment:
Is the removal of this big chunk expected?

Reply:
Same here, ignore all `data.json` changes.